application/controllers/admin/Export.php

```diff
@@ -957,64 +957,71 @@ public function exportMultipleSurveys(string $sSurveys, string $sExportType)
             if ($iSurveyID === false) {
                 continue;
             }
-            if (Permission::model()->hasSurveyPermission($iSurveyID, 'responses', 'export')) {
```

Comment:
I don't have issue with multiple ? 
It"s OK, only on single.

But seems you fix a lot more